external/js/cothority/src/skipchain/skipchain-rpc.ts

```diff
@@ -181,7 +181,7 @@ export default class SkipchainRPC {
             const last = newBlocks[newBlocks.length - 1];
             let isInRoster = false;
             for (const n of last.roster.list) {
-                if (n.getWebSocketAddress() === this.conn.getURL()) {
+                if (new URL(n.getWebSocketAddress()).href === new URL(this.conn.getURL()).href) {
```

Comment:
Why is it needed? The `URL -> string` translation is already done by `getURL` and there should not be retrocompat issue, so, that would be equivalent, no?

Reply:
true, was needed with a previous version.